initfiles/componentfiles/configschema/xsd/roxie.xsd

```diff
@@ -0,0 +1,640 @@
+                                  hpcc:presetValue="true" hpcc:tooltip="special fast-lane queue for simple queries"/>
+                    <xs:attribute name="forceStdLog" type="xs:boolean" hpcc:displayName="Force Std Log Output"
+                                  hpcc:presetValue="false"
+                                  hpcc:tooltip="Force log output to stderr even when redirected to null"/>
+                    <xs:attribute name="ignoreMissingFiles" type="xs:boolean" hpcc:displayName="Ignore Missing Files"
+                                  hpcc:presetValue="false" hpcc:tooltip="Ignore missing files"/>
+                    <xs:attribute name="indexReadChunkSize" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="Index Read Chunk Size" hpcc:presetValue="60000"
+                                  hpcc:tooltip="Break up results from indexRead (and other remote activities) every N bytes"/>
+                    <xs:attribute name="initIbytiDelay" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="Init IBYTI Delay Time (ms)" hpcc:presetValue="100"
+                                  hpcc:tooltip="Initial time (in milliseconds) a secondary agent will wait for an IBYTI packet from a primary peer"/>
+                    <xs:attribute name="jumboFrames" type="xs:boolean" hpcc:displayName="Jumbo Frames"
+                                  hpcc:presetValue="false"
+                                  hpcc:tooltip="Set to true if using jumbo frames (MTU=9000) on the network"/>
+                    <xs:attribute name="linuxYield" type="xs:boolean" hpcc:displayName="Linux Yield"
+                                  hpcc:presetValue="false"
+                                  hpcc:tooltip="Yield to scheduler in some tight loops. May help latency on uniprocessor machines"/>
+                    <xs:attribute name="maxBlockSize" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="Maximum Block Size (bytes)" hpcc:presetValue="10000000"
+                                  hpcc:tooltip="Max size of block read from client socket"/>
+                    <xs:attribute name="maxLockAttempts" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="Maximum Lock Attempts" hpcc:presetValue="5"
+                                  hpcc:tooltip="Number of retries to get lock for global queries"/>
+                    <xs:attribute name="memoryStatsInterval" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="Memory Stats Interval (s)" hpcc:presetValue="60"
+                                  hpcc:tooltip="Interval (in seconds) between reports on Roxie heap usage"/>
+                    <xs:attribute name="memTraceSizeLimit" type="xs:nonNegativeInteger"
+                                  hpcc:displayName="Memory trace Size Limit (bytes?)" hpcc:presetValue="0"
```

Comment:
Trace (capital T)?

Reply:
Human readable strings, such as display name and tooltip, will go through a separate review by the doc team. So, these are not under review for this PR.